scripts/04a-dupree-analysis.R

```diff
@@ -78,17 +89,21 @@ run_dupree_workflow <- function(local_repo, results_file, min_block_size) {
 
 # --
 
-run_workflow <- function(package, local_repo, results_dir, min_block_sizes) {
-  message("Running dupree workflow for: ", package)
+run_workflow <- function(
```

Comment:
? add package name to the results data-frame, and save the data-frame, rather than a .rds

Reply:
Done